### ISSUE TITLE
No option to record 'birds only' within 'target taxa'

### ISSUE THREAD

**Issue by Emmawilsonjncc** [ices-tools-dev/esas#145]
Can a 'birds only' option be added to 'target taxa'? This would be useful if, for example, observers have not recorded cetacean sightings during their survey. Currently, the only way 'bird only' survey data can be logged is by using the 'other' option within 'target taxa', however the description for 'other' states 'legacy data, do not use for new submissions' so I don't think we can use this either? 

Any help greatly appreciated!

**Comment by nicolasvanermen** [comment]
Yes I do think this would be a good option, and include an extra value to the vocabulary, e.g.:
19 -  Only birds, no cetaceans
... or alternatively:
19 - All species except cetaceans

However, I do wonder whether the recording of cetaceans should not be specified in the other codes, in particular code 1:
1 - All species, including cetaceans (standard)

The question is whether this actually holds true for all historical data that are currently flagged by target taxa code 1...
